client/src/main/java/com/influxdb/client/internal/InfluxQLQueryApiImpl.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.influxdb.client.internal;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.BiConsumer;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import com.influxdb.Cancellable;
+import com.influxdb.client.InfluxQLQueryApi;
+import com.influxdb.client.domain.InfluxQLQuery;
+import com.influxdb.client.service.InfluxQLQueryService;
+import com.influxdb.internal.AbstractQueryApi;
+import com.influxdb.query.InfluxQLQueryResult;
+import com.influxdb.utils.Arguments;
+
+import okhttp3.ResponseBody;
+import okio.BufferedSource;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.jetbrains.annotations.NotNull;
+import retrofit2.Call;
+
+public class InfluxQLQueryApiImpl extends AbstractQueryApi implements InfluxQLQueryApi {
+
+    private final InfluxQLQueryService service;
+
+    public InfluxQLQueryApiImpl(@Nonnull final InfluxQLQueryService service) {
+
+        Arguments.checkNotNull(service, "service");
+
+        this.service = service;
+    }
+
+    @NotNull
+    @Override
+    public InfluxQLQueryResult query(@NotNull final InfluxQLQuery influxQlQuery) {
```

Comment:
Please use `@Nonnull`:

```suggestion
    public InfluxQLQueryResult query(@Nonnull final InfluxQLQuery influxQlQuery) {
```